framework/src/st_things/things_stack/framework/things_data_manager.h

```diff
@@ -94,9 +94,9 @@ typedef struct st_device_s {
 
 size_t get_json_file_size(const char *filename);
 char *get_json_string_from_file(const char *filename);
-int dm_init_module(const char *info_Path);
 
-int dm_termiate_module(void);
+int dm_init_module(const char *info_Path);
```

Comment:
Change upper case to lower case.  for coding rule, `info_path`

Reply:
fixed.